### PR TITLE
DRYD-1036: Create Iteration Report procedure

### PR DESCRIPTION
**Pull request by mikejritter**
Jira Ticket: https://collectionspace.atlassian.net/browse/DRYD-1036

Create record type for the iterationreport.

As an additional note -- I just realized I didn't add the usual tests for the record type, I'll add them shortly.

### CODE REVIEW

File: src/plugins/recordTypes/iterationreport/fields.js [hunk 1]
```diff
@@ -0,0 +1,1109 @@
...
+          service: {
+            ns: 'http://collectionspace.org/services/iterationreport',
+          },
+        },
+        iterationIdentificationNumber: {
+          [config]: {
+            cloneable: false,
+            messages: defineMessages({
+              inUse: {
+                id: 'field.iterationreports_common.iterationIdentificationNumber.isUse',
```
Comment (ray-lee) [inline src/plugins/recordTypes/iterationreport/fields.js:48]: Typo: `isUse`

**Review by ray-lee — COMMENTED** [review]
This mostly looks good, just a few comments below.

**Review by ray-lee — COMMENTED** [review]
I think there was a misunderstanding about where to add `<Panel>` tags. I tried to be more clear, and added screenshots. Let me know if you have any questions.

File: src/plugins/recordTypes/iterationreport/forms/default.jsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+              <Row>
+                <Field name="modificationApprovalEntity" />
+                <Field name="modificationApprovalDate" />
+              </Row>
+              <Field name="modificationDescription" />
+            </Field>
+          </Field>
+        </Panel>
+
+        <Panel>
```
Comment (ray-lee) [inline src/plugins/recordTypes/iterationreport/forms/default.jsx:130]: This `<Panel>` was added outside of `<Field name="installationGroupList">`, but it should be inside `<Field name="installationGroup">`.

Current: (Note the unneccessary border and padding around "Further installation detail", and lack of padding around contents of "Further installation detail")
Screen Shot 2022-06-16 at 9 56 25 PM

Desired:
Screen Shot 2022-06-16 at 9 59 29 PM

**Comment by mikejritter** [comment]
@ray-lee Yea I thought the updated Panels looked a little awkward before but couldn't quite put my finger on why. I updated them so the padding is correct.

**Review by ray-lee — APPROVED** [review]
Looks good!

**Pull request merged**